pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -565,7 +589,15 @@ func GetImage(imageURL string) (image, imageTag, imageContext string) {
 	if len(imageParts) > 1 {
 		imageTag = imageParts[len(imageParts)-1]
 	}
-	return image, imageTag, imageContext
+	return image, imageTag, imageContext, isImageValid(imageAndTag)
+}
+
+func isImageValid(imageAndTag string) error {
```

Comment:
what is the purpose of this function if all places the GetImage func is called the error is ignored?